api/src/main/java/io/opentelemetry/trace/TraceVersion.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+public class TraceVersion {
```

Comment:
```suggestion
final class TraceVersion {
```

Reply:
it cannot be package private because accessed outside of pakcage

Reply:
You can move it to the right package though. Or just make it a member function of `HttpTraceContext`

Reply:
no, problem that I want to call `BigendianEncoding.isValidBase16String(...)` method, but class `BigendianEncoding` is package private in `io.opentelemetry.trace`.

but `HttpTraceContext` class in package `io.opentelemetry.trace.propagation`.

So I need to make TraceVersion public class in package `io.opentelemetry.trace`.
Or move class  `HttpTraceContext` to package `io.opentelemetry.trace`.

Reply:
I think I'd prefer a 3rd option, which is to create a public utility class (called maybe something like "WireFormatUtils"?) that will provide these needed methods for any propagator that needs them. This could then delegate to the BigEndianEncoding implementation, as appropriate, or maybe we move methods in there, as appropriate.

Reply:
renamed TraceVersion to WireFormatUtils with method

Reply:
removed